pkg/antctl/raw/multicluster/get/member_token.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package get
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+type tokenOptions struct {
+	namespace string
+	output    string
+	k8sClient client.Client
+}
+
+var cmdToken *cobra.Command
+
+var optionsToken *tokenOptions
+
+var tokenExamples = strings.Trim(`
+# Fetch a member token and print it in YAML format.
+  $ antctl mc get membertoken cluster-east-token -n antrea-multicluster
+# Fetch a member token and save the Secret manifest to a file.
+  $ antctl mc get membertoken cluster-east-token -n antrea-multicluster -o token-secret.yml
+`, "\n")
+
+func (o *tokenOptions) validateAndComplete(cmd *cobra.Command) error {
+	if o.namespace == "" {
+		return fmt.Errorf("Namespace is required")
+	}
+
+	var err error
+	if o.k8sClient == nil {
+		o.k8sClient, err = common.NewClient(cmd)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func NewMemberTokenCommand() *cobra.Command {
+	cmdToken = &cobra.Command{
+		Use:     "membertoken",
+		Short:   "Fetch a member token in a leader cluster",
+		Args:    cobra.MaximumNArgs(1),
+		Example: tokenExamples,
+		RunE:    runEToken,
+	}
+	o := &tokenOptions{}
+	optionsToken = o
+	cmdToken.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the token")
+	cmdToken.Flags().StringVarP(&o.output, "output", "o", "", "Output file to save the token Secret manifest")
```

Comment:
@jianjuns Hi Jianjun, I saw your MR yesterday, but I think we planned to list more than one secret then？because we want to get the Opaque secret instead of the original create secret itself

Reply:
Not sure I get your point.

In my implementation, for "get" without "--output-file", the default table output just list a subset of fields of the Secret/token, so it does not matter it is the original data or not. For json or yaml output, I feel ok to output the original Secrets too.

But if you mean you want to directly output the Secret with data directly with the json or yaml output, I am fine with that too. Then you need not the "--output-file" flag.

Reply:
> But if you mean you want to directly output the Secret with data directly with the json or yaml output, I am fine with that too. Then you need not the "--output-file" flag.

In case I did not express it clearly, I meant:

1. `antct mc get membertoken -n ns -o yaml` prints all the Secrets with opaque data in the yaml format
2. "-o json" prints the Secrets in json format.
3. `antct mc get membertoken -n ns token-name -o yaml` prints a single token Secret
4. `antct mc get membertoken -n ns [token-name]` prints the token(s) in the table format - I think only names.